### ISSUE TITLE
Do we have the ability to run a .js before all tests?

### ISSUE THREAD

**Issue by alshdavid** [DevExpress/testcafe#5513]
Is it possible for me to run test setup in a JavaScript file that's part of the same process?

I want to be able to start my http server before my tests run and have access the server reference.
The `--app` command requires an `--app-init-timeout` flag which is brittle as my CI agents aren't perfect. 

Right now I am using `forever` and `wait-on` to wait for the server before starting the tests

```shell
npx forever start -v -c ts-node ./server/index.ts
npx wait-on tcp:8080
npx testcafe chrome:headless ./tests
forever stopall
```

**Comment by AlexSkorkin** [comment]
Thank you for your inquiry. This issue looks like a question that would be best [asked on StackOverflow](https://stackoverflow.com/questions/ask?tags=testcafe) - an amazing platform for users to ask and answer questions.

We try to keep the GitHub issues tracker for bugs and feature requests only (see also: [Contributing](https://github.com/DevExpress/testcafe/#contributing)).

If you think that this issue is a bug, feel free to [open a new issue](https://github.com/DevExpress/testcafe/issues/new?template=bug-report.md), but please make sure to follow the bug template. Thank you in advance.
